### PR TITLE
Feat/controller support menu

### LINKED ISSUE #356: Change: On Hover & On Click Image for Buttons
ask elize or jeroen for the new images

### PR DESCRIPTION
**Pull request by hassanmohamed2003**
## Description
Able to navigate menus with controller

## Setting up testing environment
1. In the project window navigate to Project/Assets/Scenes/UI and open the "MainMenuUI" and "Final Exam" scene.
2. Press Play.

#### Controller support
Criteria:
- [ ] Able to complete tutorial and skip tutorial text by pressing control
- [ ] Able to go to pause screen by pressing select and navigate
- [ ] Able to navigate main menu

## Checklist
_These checks should always be true for your pull request_
- [x] My pull request is for one story/feature.
- [x] Every individual commit in this pull request is logical.
- [x] All code, documentation, commits and player seen texts are in English.
- [x] I have deleted unused / unnecessary code.
- [x] If my edits need a change in documentation, then I have updated the documentation.
- [x] All code is in correspondence with the code conventions.

### CODE REVIEW

File: Assets/Scripts/Game/GameManager.cs
Comment (KiwiLT) [inline Assets/Scripts/Game/GameManager.cs:1]: You can directly trigger TogglePause from the OnOptionsInput event, no need for an extra function

**Review by KiwiLT — CHANGES_REQUESTED** [review]
pause menu doesn't seem to work for me, can you check with me what's happening?

**Comment by ThijsvanDijk** [comment]
Can you change hover and click image to the following
Click_on_state_button
Hover_state_button

You can put a script on all buttons (search for buttons in scene with t:Button in searchbar):
Make a scriptable object that holds a reference to three sprites for hover, click and idle.
In the script swap out button sprite on select, on hover, on click

**Comment by hassanmohamed2003** [comment]
brown click, light hover

**Review by ThijsvanDijk — CHANGES_REQUESTED** [review]
Works pretty nicely overall, but a few things need fixing:
Level Select:
- [ ] Final Exam in timetable does not highlight like Assignment

Credits:
- [ ] Both selecting and clicking with controller does not work for me

Pause menu:
- [ ] Cannot close button map after opening it
- [ ] After closing an overlay in pause menu with mouse could not interact with options menu anymore

Results:
- [ ] Cannot do anything on results screen

End Cutscene:
- [ ] Cannot skip

End Certificate:
- [ ] Cannot skip

**Comment by hassanmohamed2003** [comment]
> Works pretty nicely overall, but a few things need fixing: Level Select:
> 
> * [ ]  Final Exam in timetable does not highlight like Assignment
> 
> Credits:
> 
> * [ ]  Both selecting and clicking with controller does not work for me
> 
> Pause menu:
> 
> * [ ]  Cannot close button map after opening it
> * [ ]  After closing an overlay in pause menu with mouse could not interact with options menu anymore
> 
> Results:
> 
> * [ ]  Cannot do anything on results screen
> 
> End Cutscene:
> 
> * [ ]  Cannot skip
> 
> End Certificate:
> 
> * [ ]  Cannot skip

lmao a few

**Pull request closed without merging**